### ISSUE TITLE
`--output` not working as expected

### ISSUE THREAD

**Issue by martin-braun** [jrvarma/icalcli#11]
I try to print more information in the weekly view, my command is:

```
icalcli -c "$HOME/.sync/ccommon/gf.icalcli.py" w -o l -o e -o d -o f --monday --military
```

Configuration loads an `ics` file `gf.ics` in the same directory successfully:

```
from icalcli import ICSInterface
backend_interface = ICSInterface(__file__.removesuffix(".icalcli.py") + ".ics")
```

The calendar is shown, but it is shown as if `-o l -o e -o d -o f` was never supplied. I tried individual outputs to double check. Nothing besides the time and name of the event is listed in the week view.

**Comment by jrvarma** [comment]
The `--output` options work with the agenda and search views but not in week and month views. The horizontal space available in these views is very limited and the current code prints only the event title split over as many lines as necessary. 

I will treat this as a feature request and will look into whether the event details can be split into multiple lines instead. The relevant portion is in the function `GraphEvents` beginning with the line `print_len, cut_idx = self.get_cut_index(curr_event.title)`. The idea would be to replace `curr_event.title` with a new variable `curr_event_text` built according to the `--output` options reusing the code present in the function `PrintEvent`. This function could be split into two functions the first of which returns the event details as a string and the second prints this string. `GraphEvents`  could then use the first function to create `curr_event_text`. It is doable, but requires some work.

If you submit a pull request, it will make things easier for me.

**Comment by martin-braun** [comment]
@jrvarma Fully understood and much appreciation for acting on it immediately. I just tested it and it works like charm. Obviously there isn't much space for much information, so one needs to consider increasing the width of the table view or reduce the number of output. That's a problem to solve by the user. You did a great job with this tool. It's such a bless to work with ICS files using your tool.

Thank you! :)

**Comment by martin-braun** [comment]
@jrvarma I increased the width in my above example and notice that when I do so, some padding is off after the start and end time tho:

<img width="314" alt="image" src="https://github.com/jrvarma/icalcli/assets/24259245/95d83298-6323-42c9-8c6d-61e498fbe3db">

```
icalcli -c "$HOME/.sync/ccommon/gf.icalcli.py" w -w 15 -o l -o e -o d -o f --monday --military
```

It seems to happen only after the start/end time.

**Comment by jrvarma** [comment]
> @jrvarma I increased the width in my above example and notice that when I do so, some padding is off after the start and end time tho:

**Comment by jrvarma** [comment]
> Hi @jrvarma I also spotted another bug. -o l -o e -o d -o f will output the alarm and not the description. I believe there is just a copy-paste issue and d outputs the alarm information instead of the description.

This is also hopefully fixed in version 1.0.6. Description was being ignored earlier. Alarm is being printed by default unless you pass `-o noa` as an option.

**Comment by martin-braun** [comment]
@jrvarma I can confirm those issues being fixed, good job! :)
